src/ui/bar-chart/BarChart.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactNode, useEffect, useState, Children } from 'react';
+
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+
+import csx from "./BarChart.scss"
+
+interface IBarChart {
+    children: ReactNode[] | ReactNode;
+    title: string
+}
+
+const BARS_LIMIT = 10;
+
+const BarChart = ({ children, title }: IBarChart) => {
+    const [width, setWidth] = useState<number>(0);
+    const [bars, setBars] = useState<typeof children | typeof children[]>([]);
+    const [currentPage, setCurrentPage] = useState<number>(1);
```

Comment:
Type interference will handle types in useState hooks - delete <number> not needed

Reply:
done